.scalafmt.conf

```diff
@@ -3,3 +3,4 @@ runner.dialect = scala213
 align.preset = more
 maxColumn = 120
 rewrite.rules = [AsciiSortImports,AvoidInfix,SortModifiers]
+project.layout = StandardConvention
```

Comment:
This is necessary because we now have a `scala-3` folder and this config option allows scalafmt to automatically pick up dialect based on folder structure.